src/plugin/sdl/sdl.c

```diff
@@ -225,7 +191,88 @@ int SDL_priv_init(void)
   return 0;
 }
 
-int SDL_init(void)
+static void SDL_text_init(void)
+#if defined(HAVE_SDL2_TTF) && defined(HAVE_FONTCONFIG)
+{
+  int err;
+  char *pth = NULL;
+  FcPattern *pat, *match;
+  FcResult result;
+  char *foundname;
+
+  use_bitmap_font = 1;
```

Comment:
Why?

Reply:
Default to vga unless we successfully setup ttf.